contrib/fasttext/patches/no-march-native.patch

```diff
@@ -0,0 +1,24 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -15,7 +15,7 @@
+ 
+ include_directories(fasttext)
+ 
+-set(CMAKE_CXX_FLAGS " -pthread -std=c++11 -funroll-loops -O3 -march=native")
++set(CMAKE_CXX_FLAGS " -pthread -std=c++11 -funroll-loops -O3")
+ 
+ set(HEADER_FILES
+     src/args.h
+--- a/setup.py
++++ b/setup.py
+@@ -75,8 +75,8 @@
+             FASTTEXT_SRC,
+         ],
+         language='c++',
+-        extra_compile_args=["-O0 -fno-inline -fprofile-arcs -pthread -march=native" if coverage else
+-                            "-O3 -funroll-loops -pthread -march=native"],
++        extra_compile_args=["-O0 -fno-inline -fprofile-arcs -pthread" if coverage else
```

Comment:
these should *all* be patched out

Reply:
okay

Reply:
do the flags in CMakelists.txt need patching out as well?

Reply:
yes, all but perhaps `-std=c++11`

Reply:
okay